### PR TITLE
chore(sdk): add `absl-py` in kfp install_requires

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**
We have SDK code relying on this, for example:
https://github.com/kubeflow/pipelines/blob/279694ec6d4d0467bef26380ebdae37c487518c2/sdk/python/kfp/dsl/artifact.py#L17

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: sdk/python/setup.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #VERSION = .... Change the version in kfp/__init__.py
 
 REQUIRES = [
+    'absl-py',
```
Comment (numerology) [inline sdk/python/setup.py:23]: Maybe we should limit the version as well?

**Review by numerology — COMMENTED** [review]
Thanks for the fix!

File: sdk/python/setup.py (hunk 1 above)
Comment (chensun) [reply sdk/python/setup.py:23]: Good point. 

Just for reference, TFX currently sets: [`'absl-py>=0.9,<0.11'`](https://github.com/tensorflow/tfx/blob/77acc3ff231191597fead9a3f246a96294a883e1/tfx/dependencies.py#L55), while [0.11 was out in October 2020](https://pypi.org/project/absl-py/0.11.0/)

So how about we do `absl-py>=0.9,<=0.11`?

Comment (numerology) [reply sdk/python/setup.py]: SGTM.

**Pull request merged**